feature/interests/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/interests/InterestsItem.kt

```diff
@@ -83,7 +84,11 @@ fun InterestsItem(
             )
         },
         colors = ListItemDefaults.colors(
-            containerColor = Color.Transparent,
+            containerColor = if (isSelected) {
```

Comment:
Instead of just changing the container color, we should also update the semantics to reflect being selected for accessibility.